display/ssd1306.h

```diff
@@ -445,69 +461,118 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
     }
   }
 
+  void SB_On2() override {
+    if (IMG_out) {
+      SetFile(&IMG_out, round(Set_Duration(font_config.ProffieOSOutImageDuration)));
+    }
+  }
+
   void usb_connected() override {
     if (EscapeIdleIfNeeded() && current_effect_ == &IMG_idle) {
       // We are idle-looping, and usb is connected. Time to stop.
-      SetMessage("usb");
+      SetMessage("    usb\nconnected");
       SetScreenNow(SCREEN_MESSAGE);
     }
   }
 
-  void SB_Effect(EffectType effect, float location) override {
+  float Set_Duration(float image_duration) {
```

Comment:
This is a weird naming conversion.
The two "normal" ones are:

   snakeCaseNamingConvention

and

   underscore_naming_convention

Underscores *and* capital letters is a bit unusual.

Also, I think it would be better to have a helper function which wraps ShowFile.
Maybe something like:

void SHowFileWithSoundLength(Effect* e, float duration) { ... }

Reply:
ah.
Ok, I changed it to SetDuration() for the naming convention.
I'll need to work on how to incorporate ShowFile.

Reply:
> Also, I think it would be better to have a helper function which wraps ShowFile.

Ok, edited.